### PR TITLE
Create model factories

### PR DESCRIPTION
**Pull request by elguitarraverde**
We have created most of the model factories so that we can seed the database with data.

### CODE REVIEW

File: database/seeders/DatabaseSeeder.php [hunk 1]
```diff
@@ -23,5 +37,20 @@ public function run()
         $this->call(RoleSeeder::class);
         $this->call(UserSeeder::class);
         $this->call(IndustrySeeder::class);
+
```
Comment (gnovaro) [inline database/seeders/DatabaseSeeder.php:40]: Should be useful to check the configuration env variable for if you want to install with demo data or not. If a production installation we need a clean database.

Comment (gnovaro) [reply database/seeders/DatabaseSeeder.php:40]: if ( !app()->environment('production'))
{
        Account::factory(20)->create();
        Brand::factory(20)->create();
        Category::factory(20)->create();
        Industry::factory(20)->create();
        Lead::factory(20)->create();
        Customer::factory(20)->create();
        Contact::factory(20)->create();
        EmailTemplate::factory(20)->create();
        Email::factory(20)->create();
        Order::factory(20)->create();
        Product::factory(20)->create();
        Item::factory(20)->create();
        Supplier::factory(20)->create();
        Ticket::factory(20)->create();
}

**Review by gnovaro — CHANGES_REQUESTED** [review]
Please check my comment, thanks for your work. We will include TDD test cases in the future

File: database/seeders/DatabaseSeeder.php (hunk 1 above)
Comment (elguitarraverde) [reply database/seeders/DatabaseSeeder.php]: good idea!!

https://github.com/Roskus/hammercrm/pull/34/commits/eec15c2ac87fa0c716fb8075b83e54543ba7c2fe
In this commit I have separated the seeders according to the environment.

I hope it is what you were looking for.

I can get started with the tests if you want.

**Comment by elguitarraverde** [comment]
We will be making more atomic commits for Factories in the future.

**Pull request closed without merging**